src/client/sandbox/event/listeners.js

```diff
@@ -68,7 +68,7 @@ export default class Listeners extends EventEmitter {
     }
 
     _createEventHandler () {
-        var listeningCtx = this.listeningCtx;
+        var listeners = this;
 
         return function (e) {
```

Comment:
I removed `eslint-disable comment` in two places:
https://github.com/miherlosev/testcafe-hammerhead/blob/i_292/src/client/sandbox/code-instrumentation/properties/index.js#L448
https://github.com/miherlosev/testcafe-hammerhead/blob/i_292/src/client/sandbox/code-instrumentation/properties/index.js#L454